.claude/CLAUDE.md

```
@@ -0,0 +1,1366 @@
+    
+    def __init__(self, service: str, message: str):
+        super().__init__(
+            f"{service} error: {message}",
+            "{MODULE}_EXTERNAL_ERROR"
+        )
+        self.service = service
+```
+
+### Exception Hierarchy and HTTP Mapping
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)